lib/changeNIDMtoxSPM.m

```diff
@@ -63,7 +83,10 @@
     %M
     
     %Locate the excursion set maps and their indices in the graph.
-    [excursionSetMaps, excurIndices] = searchforType('nidm_ExcursionSetMap', graph);
+    [excursionSetMaps, ~] = searchforType('nidm_ExcursionSetMap', graph);
```

Comment:
The map here takes the ID of a node and returns integers corresponding to which excursionsets link to the node.

If we were to save the type of each object inside the hashmap also, that would mean to do what the above code does I assume we would have to traverse the list of keys (the node ID's), look inside the hashmap for each ID and keep only the ID's of the nodes of the type we want. This does seem more efficient.

The problem would then be however, that we would have to use searchforID on the ID's returned and traverse the graph to obtain the actual nodes themselves from the ID's and then we have done the same amount of computation. 

Therefore I don't think it would be any different in time efficiency. Unless I have missed something...